chainer/training/extensions/log_report.py

```diff
@@ -56,8 +56,17 @@ class LogReport(extension.Extension):
             formatting. For example, users can use '{iteration}' to separate
             the log files for different iterations. If the log name is None, it
             does not output the log to any file.
+            It is recommended that you explicitly configure the filename like
+            ``'log.jsonl'`` when ``file_format='jsonl'`` to avoid confusing
+            third-party software which assume ``log`` file has a default single
+            JSON list format.
             For historical reasons ``log_name`` is also accepted as an alias
             of this argument.
+        file_format (str): Format of the log file. If ``'jsonl'`` is specified,
+            a log file is in JSON Lines format, which is JSON objects delimited
+            by newlines (LF). It is suitable for stream processing and works
```

Comment:
Shouldn't it include `CR`s in Windows, as in the case of JSON?

Reply:
I think `LF` in JSONL is just a separator, not a human-readable symbol. The [guideline](http://jsonlines.org/) states that _"Line Separator is `\n`"_. It also accepts `\r\n` but `CR` will be simply ignored by the JSON parsers as trailing whitespace.